### DISCUSSION TITLE
CLI notation: eg. +player/inventory=hard_mode

### DISCUSSION THREAD

**Discussion by francisco-camargo** [mit-ll-responsible-ai/hydra-zen#408] [Q&A] [answered]
Hey all,
[This](https://mit-ll-responsible-ai.github.io/hydra-zen/tutorials/config_groups.html#running-our-application) tutorial section introduced some notation that I don't understand
```shell
$ python my_app.py player.name=ivy +player/inventory=hard_mode
ivy, lvl: 1, has: {'gold': 0, 'weapon': 'inner thoughts', 'costume': 'rags'}
```
Why is the `+` used? It looks like it is used only when a subgroup default is used, is that correct?

Thanks,
Francisco

**Comment by jgbos** [comment, marked as answer]
Hi @francisco-camargo, the CLI is 100% Hydra here.  Be sure to familiarize yourself with Hydra CLI syntax:
  - Override Grammar: https://hydra.cc/docs/advanced/override_grammar/basic/ and https://hydra.cc/docs/advanced/override_grammar/extended/
  - CLI flags: https://hydra.cc/docs/advanced/hydra-command-line-flags/
  - Defaults List: https://hydra.cc/docs/advanced/defaults_list/
  
I think we should definitely add a page in the docs referencing these links.

**Reply by francisco-camargo** [reply]
Got it, that looks like exactly what I was hoping to see, thanks!
